pkg/util/cloud.go

```diff
@@ -62,3 +63,43 @@ func GetPodNamespace(annotations map[string]string) string {
 
 	return annotations[cri.SandboxNamespace]
 }
+
+// Method to get instance type from annotations
+func GetInstanceTypeFromAnnotation(annotations map[string]string) string {
+	return annotations[hypannotations.MachineType]
+}
+
+// Method to check if a string exists in a slice
+func Contains(slice []string, s string) bool {
+	for _, item := range slice {
+		if item == s {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Can we use the `slices` package? This package will be a part of the standard Go library in Go 1.21.
https://pkg.go.dev/golang.org/x/exp/slices#Contains